pyproject.toml

```diff
@@ -47,7 +47,8 @@ influxdb = "^5.3.1"
 # required for tests to pass right now. There are often issues when pinning to a branch
 # and calling poetry update. I have had to completely remove my venv and reinstall in order
 # to get the latest updates.
-alfalfa-client = "~0.5.0"
+alfalfa-client = { git = "https://github.com/nrel/alfalfa-client.git", branch = "handle_none_output_values" }
```

Comment:
you meant to commit this?

Reply:
yeah... That's actually this PR in the client which is required to run the tests https://github.com/NREL/alfalfa-client/pull/56